### ISSUE TITLE
Error running on clean server, missing community enablement

### ISSUE THREAD

**Issue by hskrtich** [capitanh/microk8s_ansible_role#1]
I ran in to an error when trying to run your role on a fresh server.  I manually ran `microk8s enable community` and that seems to have fixed this issue.

```
failed: [10.10.10.10] (item=traefik) => {"ansible_loop_var": "item", "changed": true, "cmd": ["microk8s", "enable", "traefik"], "delta": "0:00:00.324848", "end": "2024-03-28 05:14:43.835459", "item": "traefik", "msg": "non-zero return code", "rc": 1, "start": "2024-03-28 05:14:43.510611", "stderr": "Addon traefik was not found in any repository", "stderr_lines": ["Addon traefik was not found in any repository"], "stdout": "To use the community maintained flavor enable the respective repository:\n\n    microk8s enable community", "stdout_lines": ["To use the community maintained flavor enable the respective repository:", "", "    microk8s enable community"]}
changed: [10.10.10.10] => (item=registry)
failed: [10.10.10.10] (item=linkerd) => {"ansible_loop_var": "item", "changed": true, "cmd": ["microk8s", "enable", "linkerd"], "delta": "0:00:00.341410", "end": "2024-03-28 05:14:53.293612", "item": "linkerd", "msg": "non-zero return code", "rc": 1, "start": "2024-03-28 05:14:52.952202", "stderr": "Addon linkerd was not found in any repository", "stderr_lines": ["Addon linkerd was not found in any repository"], "stdout": "To use the community maintained flavor enable the respective repository:\n\n    microk8s enable community", "stdout_lines": ["To use the community maintained flavor enable the respective repository:", "", "    microk8s enable community"]}
failed: [10.10.10.10] (item=istio) => {"ansible_loop_var": "item", "changed": true, "cmd": ["microk8s", "enable", "istio"], "delta": "0:00:00.317950", "end": "2024-03-28 05:14:57.436015", "item": "istio", "msg": "non-zero return code", "rc": 1, "start": "2024-03-28 05:14:57.118065", "stderr": "Addon istio was not found in any repository", "stderr_lines": ["Addon istio was not found in any repository"], "stdout": "To use the community maintained flavor enable the respective repository:\n\n    microk8s enable community", "stdout_lines": ["To use the community maintained flavor enable the respective repository:", "", "    microk8s enable community"]}
```

**Comment by capitanh** [comment]
Thanks for the tip. Out of curiosity, which OS is this fresh server running?

**Comment by capitanh** [comment]
Fixed. Thanks again!

**Comment by hskrtich** [comment]
Thanks for the fix! Its a ubuntu 22.04 box tho I am not sure what all the VPS provider changed on the image they use. Its already had some weird issues around not having snapd installed.
